source/localizable/testing/index.md

```diff
@@ -44,7 +44,7 @@ in ensuring data and actions are properly passed between different parts of the
 provide confidence that parts of the system will work within the application under multiple
 scenarios.
 
-It is recommended that components be tested with integration tests because the component
+It is recommended that components and helpers be tested with integration tests because the component
```

Comment:
Since we added `helpers` here, we should probably update the end of this line from `because the component interacts with the system` to `because they both interact with the system`